lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -162,8 +162,10 @@ external JsFlutterConfiguration? get _jsConfiguration;
 
 /// The JS bindings for the object that's set as `window.flutterConfiguration`.
 @JS()
-@anonymous
```

Comment:
FYI (no action needed): You can technically keep the `@anonymous` here. The only real difference nowadays with `@JS` and `@JS` + `@anonymous` is the former's constructors bind to some specific type, while the latter's creates an object literal. Everything else is semantically the same. It's a bit verbose but adding three annotations works fine. If there is no need to create a literal (which the lack of a factory indicates is the case), I like removing the `@anonymous`.

Reply:
Okay, I will keep this in mind. Thanks!